rkt/rkt.go

```diff
@@ -221,7 +228,9 @@ func main() {
 	// cmdRkt.GenBashCompletionFile("dist/bash_completion/rkt.bash")
 	// os.Exit(0)
 
-	cmdRkt.Execute()
+	if err := cmdRkt.Execute(); err != nil && cmdExitCode == 0 {
```

Comment:
Is there something more specific we can check error against? might there not be false positives here?